.gitignore

```diff
@@ -0,0 +1,24 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/front/node_modules
+/.pnp
+.pnp.js
+
```

Comment:
Mb add .idea folder for ide like WebStorm?

Reply:
done